kvbc/tools/db_editor/include/kv_blockchain_db_editor.hpp

```diff
@@ -19,13 +19,18 @@
 #include "execution_data.cmf.hpp"
 #include "keys_and_signatures.cmf.hpp"
 #include "db_interfaces.h"
-#include "Crypto.hpp"
 #include "kvbc_key_types.h"
 #include "categorization/db_categories.h"
 #include "storage/merkle_tree_key_manipulator.h"
 #include "bcstatetransfer/DBDataStore.hpp"
 #include "bftengine/PersistentStorageImp.hpp"
 #include "bftengine/DbMetadataStorage.hpp"
+#include "crypto_utils.hpp"
+
+#pragma GCC diagnostic push
```

Comment:
shouldn't we do it in crypto_utils.hpp?

Reply:
no, crypto_uils.hpp is agnostic to the implementation. The implementation is in crypto_utils.cpp

Reply:
so why is there include for crypto/dll.h?